config/locales/devise.en.yml

```diff
@@ -17,7 +17,10 @@ en:
       unauthenticated: "You need to sign in or sign up before continuing."
       unconfirmed: "You have to confirm your email address before continuing."
       user:
-        invalid_login: "Invalid %{authentication_keys} or password."
+        ldap_failed: "The LDAP-support is not enabled."
+        ldap_bind_failed: "Invalid %{authentication_keys} or password."
+        invalid_username: "Invalid username. The generation of the personal namespace failed."
```

Comment:
This message is misleading since this is not necessarily the case. I'd say something like "User could not be registered. Invalid username.".